models/general/knowledge/en/founder.txt

```diff
@@ -0,0 +1,12 @@
+# tell about the founder of famous tech companies
+founder of *| who is the founder of * ?|Name the founder of * ?| Who is the father of * ?|can you tell me who is the founder of * ?| * is founded by whom ?
```

Comment:
Add these questions as well. 
Who is founder of *
Name of founder of *
what is the name of founder of *
what is name of founder of *
Name of founder of *
Tell me about founder of *
just with star ( * founder )